.cloudbuild/integration.yaml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://json.schemastore.org/cloudbuild.json
+availableSecrets:
+  secretManager:
+    - versionName: projects/ravelin-builds/secrets/ci-browserstack-token/versions/latest
+      env: BROWSERSTACK_ACCESS_KEY
+    - versionName: projects/ravelin-builds/secrets/ci-browserstack-username/versions/latest
+      env: BROWSERSTACK_USERNAME
+    - versionName: projects/ravelin-builds/secrets/ci-ngrok-token/versions/latest
+      env: NGROK_AUTH_TOKEN
+
+logsBucket: gs://ravelin-cloudbuild-logs
+options:
+  logging: GCS_ONLY
+
+steps:
+  - id: npm_install
+    name: eu.gcr.io/ravelin-builds/chromium:10.24
```

Comment:
What is this image? Where is it defined? Why are we using an image called "chromium" to run node? Why are we using an image called "chromium" when we're not using chromium?

Reply:
You're right we'll only use this image for headless chrome stuff, I've replaced the image. We could use this to run `npm run test:unit` too in the CI build. Up to you if you want to do that or if you consider the browsertack integration tests to be sufficient here.

For your information to answer your question, the image is built here: https://github.com/unravelin/custom-images/blob/main/config.yaml#L2-L9
Dockerfile is here: https://github.com/unravelin/custom-images/blob/main/images/chromium/Dockerfile